src/types/table.ts

```diff
@@ -27,7 +27,10 @@ export type TableColumnSlotName = `column-${string}`
 
 export type SwrvState = 'VALIDATING' | 'VALIDATING_HAS_DATA' | 'PENDING' | 'SUCCESS' | 'SUCCESS_HAS_DATA' | 'ERROR' | 'STALE_IF_ERROR'
 
+export type EmitState = 'loading' | 'error' | 'success'
```

Comment:
Can you please rename this to `TableState` so that it's scoped to the component? 

Also, don't you need to add types for KCatalog as well?

Reply:
I'll do the rename, but your second question is something I wanted to ask about. The way we do the imports from `/types` we aren't allowed to have any name duplication for the `type`s we define, since both `SwrvState` and `EmitState` are an exact match between the 2 components is there a need to define them twice with slightly different names?

Reply:
It depends if they always match really. Since we know we're going to refactor KTable at some point prob makes sense to name them `TableSwrvState` etc.

Reply:
Switched to `TableState`